travis/samples/ngx-h2mem-social-ws-nol2/.yo-rc.json

```diff
@@ -4,13 +4,13 @@
     "baseName": "travisSocialWebsocket",
     "packageName": "io.github.jhipster.travis",
     "packageFolder": "io/github/jhipster/travis",
-    "authenticationType": "session",
-    "cacheProvider": "ehcache",
-    "enableHibernateCache": true,
+    "authenticationType": "jwt",
+    "cacheProvider": "no",
```

Comment:
this is a no cache test and not nol2

Reply:
you're right, I was wrong. I will change it

Reply:
suggest to just enable cache here and keep l2 off